src/modules/authentacation/components/ChangePassword/ChangePassword.tsx

```diff
@@ -1,7 +1,129 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { useForm } from 'react-hook-form'
+import { PASSWORD_VALIDATION } from '../../../../services/validation/validation'
+import { axiosInstance, USERS_URLS } from '../../../../services/apisUrls/apisUrls'
+import { useNavigate } from 'react-router-dom'
+import { toast } from 'react-toastify'
+import { FaRegEyeSlash, FaSpinner } from 'react-icons/fa'
+import { IoEyeOutline } from 'react-icons/io5'
+
+interface formData {
+  oldPassword: string,
+  newPassword: string,
+  confirmNewPassword: string
+}
 
 export default function ChangePassword() {
+  const [isPasswordVisable, setIsPasswordVisable] = useState(false);
+  const [isOldPasswordVisable, setIsOldPasswordVisable] = useState(false);
+  const [isConfirmPasswordVisable, setIsConfirmPasswordVisable] = useState(false);
+
+  const navigate = useNavigate()
+  const { register, formState: { errors, isSubmitting }, handleSubmit }
+    = useForm<formData>()
+
+  const onSubmit = async (data: any) => {
+    try {
+      let response = await axiosInstance.put(USERS_URLS.CHANGE_PASSWORD, data)
+      console.log(response);
+      navigate("/login")
+      toast.success("password updated succeffully")
+    } catch (error) {
+      console.log(error);
+      toast.error(response.data.error)
+    }
+  }
   return (
-    <div>ChangePassword</div>
+    <div>
+      <div className="my-10">
+        <h6 className="text-white">Welcome to PMS</h6>
+        <h2 className="text-primary font-bold text-3xl tracking-wide">
+          <span className="capitalize border-b-4 border-primary">C</span>hange password
+        </h2>
+      </div>
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <div className='relative'>
+          <label className="block text-primary ">Old Password</label>
```

Comment:
I suggest creating a password input component and re-use it instead of repeating the same logic multiple times.